Fix Markdown editor

kazuhitoyokoi opened the pull request:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes
Current Markdown editor has two troubles. When the Node-RED user expands the editor,  the user cannot utilize the large resolution of the display effectively because the maximum size of the editor is 1000px. Therefore, I removed the definition of the max size. After fixing it, the user experience will be the same as JavaScript editor in the function node.
And there's an unknown vertical gray line (I found that it is a print margin while fixing it.) on the editors. I added ace editor option to remove it.

![image](https://user-images.githubusercontent.com/20310935/64842520-648fb700-d63d-11e9-83c1-08471e0cc956.png)

## Checklist
- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

kazuhitoyokoi commented:
@knolleary I updated it!

knolleary commented:
This was a deliberate design choice as very long lines of text can be hard to read. I also considered the fact that often this is used to edit a node's documentation which is displayed in the limited-width sidebar.

I had also left the print margin in place to again help the user keep their line lengths to a sensible length.

But this is a very English/Latin view of what is a comfortable writing experience. So whilst I don't consider this a 'fix', I'm happy to discuss this change in the design, but would like to get some more points of view on it.

@dceejay @HiroyasuNishiyama any thoughts?

dceejay commented:
IMHO - I would leave the margin to give users who don't want to roughly split lines (though I realise that the process of going from md to html will format lines as they render in the browser). Not fussed about the width - can't really see a reason to limit it if the user has expanded the screen to max width.

HiroyasuNishiyama commented:
I don't think there are any problems specific to Japanese or other multibyte characters.
Personally, I don't like horizontal scrolling after switching to preview mode, so so I prefer splitting lines into narrower pieces.

kazuhitoyokoi commented:
Thank you for your discussions. In terms of Japanese sentences in Markdown, some people tend to write one paragraph (or one sentence) in one line because line breaks are converted to white spaces in Markdown processing as the following screenshot. Therefore, they need the wide Markdown editor. I think that the expanded Markdown editor is suitable to fill their needs.

![image](https://user-images.githubusercontent.com/20310935/65035721-d725cd00-d984-11e9-9112-71b4d5d88cec.png)

I understand the print margin policy from your comments. It was just originated from my misunderstood concern of gray line bug. So I removed the commits from my pull request.
And I also agree with avoiding horizontal scrolling in Markdown. Therefore, I added a commit to enable wrap mode for Markdown editor.

dceejay commented:
I'm ok with these changes.
